onnxconverter_common/onnx_ops.py

```diff
@@ -778,6 +778,13 @@ def apply_relu6(scope, input_name, output_name, container, operator_name=None, d
     apply_clip(scope, name_relu, output_name, container, name_relu_op + '_clip', value_6, value_0)
 
 
+def apply_relu_6(scope, input_name, output_name, container, operator_name=None, zero_value=0):
```

Comment:
in this case, zero by default will be a integer, can you change it to be a floating value?